pkg/controller/cincinnati/cincinnati_controller_test.go

```diff
@@ -139,3 +398,50 @@ func newDefaultCincinnati() *cv1alpha1.Cincinnati {
 		},
 	}
 }
+
+func newImage() *configv1.Image {
+	return &configv1.Image{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: defaults.ImageConfigName,
+		},
+		Spec: configv1.ImageSpec{
+			AdditionalTrustedCA: configv1.ConfigMapNameReference{
+				Name: testConfigMap,
+			},
+		},
+	}
+}
+
+func newConfigMap() *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      testConfigMap,
+			Namespace: openshiftConfigNamespace,
+		},
+		Data: map[string]string{
+			"testData": "some random text",
+		},
+	}
+}
+
+func verifyConditions(t *testing.T, expectedConditions []conditionsv1.Condition, cincinnati *cv1alpha1.Cincinnati) {
+	for _, condition := range expectedConditions {
+		assert.True(t, conditionsv1.IsStatusConditionPresentAndEqual(cincinnati.Status.Conditions, condition.Type, condition.Status))
+	}
+	assert.Equal(t, len(expectedConditions), len(cincinnati.Status.Conditions))
+}
+
+func verifyOwnerReference(t *testing.T, ownerReference metav1.OwnerReference, cincinnati *cv1alpha1.Cincinnati) {
+	assert.Equal(t, ownerReference.Name, cincinnati.Name)
+	//Note: These properties seem to be derived from runtime object and do not seem to match the expected values.
```

Comment:
Is this commented code here on purpose? Can it be removed?

Reply:
I'd like to leave it there, in case it trips up someone else too. I need to dig into it further and may be file a bug.